docs/achievements.md

```diff
@@ -104,7 +104,7 @@ achievements.forceSaveOnGrantOrRevoke = true -- Defaults to false. Only set if y
 | `id`                   | `string`  | Uniquely identifies the achievement. **Required.**                                                                                                |
 | `name`                 | `string`  | Nicely formatted name for the achievement. Shown to the player. **Required.**                                                                     |
 | `description`          | `string`  | Nicely formatted description for the achievement. Shown to the player. **Required.**                                                              |
-| `descriptionLocked`    | `string?` | Nicely formatted description for the achievement shown until it is granted. Shown the the player. Defaults to `nil`.                              |
+| `descriptionLocked`    | `string?` | Nicely formatted description for the achievement shown until it is granted. Shown to the player. Defaults to `nil`.                              |
```

Comment:
I mean technically it defaults to whatever "description" is, no?

Reply:
True, okay, will make the docs even more explicit on this point…

Reply:
Updated again. It now reads:
> Nicely formatted description for the achievement shown until it is granted. Shown to the player. Defaults to `nil`. When left `nil`, the value provided for `description` will be shown in both locked and unlocked states.